examples/docking-protein-protein/docking-protein-protein-cltsel-test.cfg

```diff
@@ -31,25 +31,30 @@ hise_1 = 15
 [rigidbody]
 tolerance = 20
 ambig_fname = "data/e2a-hpr_air.tbl"
-sampling = 20
+sampling = 25
+
+[caprieval]
+#tolerance = 20
+reference_fname = "data/e2a-hpr_1GGR.pdb"
 
 [clustfcc]
 #tolerance = 20
-threshold=1
+threshold = 4
 
 [seletopclusts]
 #tolerance = 20
 ## select the best 4 clusters
 # top_cluster = [1, 2, 3, 4]
 ## select all the clusters
-top_cluster = [1,2,3,4,5,6,7,8,8,10]
+top_cluster = [1, 2, 3, 4]
```

Comment:
Why not simply: `top_cluster = []` since it comes after the comment select all the clusters...